.github/workflows/linters.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.stylelintrc.json
       - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
```

Comment:
- [ ] I kindly suggest that you please don't edit you linters workflow file as it can lead to your project getting invalidated

Reply:
Thanks for pointing that out.